src/coronabot.py

```diff
@@ -33,12 +33,12 @@ def format_field(self, value, spec):
 def pretty_printed_string(json_list):
     fmt = PartialFormatter()
     r = "```"
-    r += fmt.format("|{:>20}|{:>10}|{:>10}|{:>10}|\n", "Country", "Confirmed", "Deaths", "Recovered")
+    r += fmt.format("|{:>20}|{:>10}|{:>10}|{:>10}|\n", "Country", "Confirmed", "Deaths", "ImNotDead")
     r += fmt.format("|{:>20}|{:>10}|{:>10}|{:>10}|\n", "", "", "", "")
     for item in json_list:
         thing = item['attributes']
         r += fmt.format("|{:>20}|{:>10}|{:>10}|{:>10}|\n", thing['Country_Region'], thing['Confirmed'],
-                        thing['Deaths'], thing['Recovered'])
+                        thing['Deaths'], thing['ImNotDead'])
```

Comment:
This line is not right. The header change above is all that's necessary to get what you're looking for. Remove this and I'll merge